### ISSUE TITLE
Multiple bboxes, positive and negative bboxes?

### ISSUE THREAD

**Issue by ahundt** [tnikolla/grasp-detection#14]
It seems only the positive bounding boxes might be loaded, is this the case?
![bounding box](https://user-images.githubusercontent.com/34765938/35213959-75afc960-ff9a-11e7-9916-ac11117415c5.png)

Also, how are the varying numbers of bboxes handled / structured?

The code for that part is split in several locations so it isn't easy to tell the design.

**Comment by ahundt** [comment]
Additional bug: There are also [NaN](https://en.wikipedia.org/wiki/NaN) bboxes, which aren't handled correctly.

**Comment by jinhuan-hit** [comment]
@ahundt Yeah,I also find the problem.As the issue #20 ,I have deleted the Nan valude in pcd0132cpos.txt and pcd0165cpos.txt file and run build_cgd_dataset.py.However,when I run grasp_det.py for train,the loss is nan.

**Comment by ahundt** [comment]
There are newer repositories with better results and more complete implementations available on github. It was a great starting point at the time of release, but I don't recommend using this repo/code anymore.

**Comment by jinhuan-hit** [comment]
@ahundt Thank you for your reply.

**Comment by jinhuan-hit** [comment]
@ahundt By the way,would you mind sharing the link of newer repositories with better results and more complete implementations available on github?
Thanks a lot.

**Comment by ahundt** [comment]
This is mine and has a keras/tf loader, but it only supports single grasp:
https://github.com/jhu-lcsr/costar_plan

I'm finding this pytorch implementation is pretty decent and runs on a real robot:
https://github.com/andyzeng/visual-pushing-grasping

**Comment by jinhuan-hit** [comment]
@ahundt Thanks a lot.Yeah,I have read your paper "The CoSTAR Block Stacking Dataset:
Learning with Workspace Constraints",a very interseting work about block stacking.
Andyzeng uses Reinforcement Learning to learn how to pick and place.However,the environment is limited.Thank you for your kindness.

**Comment by ahundt** [comment]
Thanks!
